source/agora/consensus/ValidatorSet.d

```diff
@@ -310,7 +262,7 @@ public class ValidatorSet
         {
             () @trusted {
                 auto results = this.db.execute("SELECT val FROM validator_set" ~
-                    " WHERE enrolled_height is not null");
+                    " ORDER BY key ASC");
```

Comment:
> Get all the current validators

This change requires the document to be changed.
The meaning of sorted should be added.